app/src/main/res/drawable-v23/splash_page.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:drawable="@color/oppiaPrimaryLight" />
+  <item
+    android:width="@dimen/splash_screen_logo_width"
+    android:height="@dimen/splash_screen_logo_height"
+    android:drawable="@drawable/oppia_logo"
```

Comment:
Could we maybe rename this to be 'oppia_logo_svg' and the png version 'oppia_logo_bitmap' to contrast?

Reply:
Done